analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ProgramStateTest.OperationValue.cs

```diff
@@ -125,6 +125,19 @@ public void SetOperationValue_WithWrapper_Overwrites()
             sut[operation].Should().Be(value2);
         }
 
+        [TestMethod]
+        public void SetOperationValue_NullValue_RemovesSymbol()
+        {
+            var value1 = new SymbolicValue(new SymbolicValueCounter());
```

Comment:
We don't have more than one. The `SymbolicValueCounter` can be removed.
```suggestion
            var value = new SymbolicValue(new());
```